addons/web/static/src/views/graph/graph_model.js

```diff
@@ -442,9 +445,42 @@ export class GraphModel extends Model {
                             lazy: false, // what is this thing???
                             context: { fill_temporal: true, ...this.searchParams.context },
                         }
-                    )
-                    .then((data) => {
+                ).then((data) => {
+                    // Compute cumulated start if needed
+                    if (cumulated_start && sequential_field && data && data.groups.length &&
```

Comment:
data is always an object with a group key

Reply:
you should activate the tooling. Type addons/web/tooling/enable.sh. This will automatically lint the code